tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -186,6 +188,10 @@ var _ = ginkgo.Describe("VM backup and restore tests", ginkgo.Ordered, func() {
 		gomega.Expect(err).To(gomega.BeNil())
 
 		dpaCR.VeleroDefaultPlugins = append(dpaCR.VeleroDefaultPlugins, v1alpha1.DefaultPluginKubeVirt)
+
+		storageClassYaml := fmt.Sprintf("./sample-applications/storageclass-immediate/%s-immediate.yaml", provider)
```

Comment:
Where are these files coming from?
https://github.com/search?q=repo%3Aopenshift%2Foadp-operator%20storageclass-immediate&type=code

Reply:
Thanks, I just forgot to add them.